src/display/EditorDisplay.java

```diff
@@ -0,0 +1,50 @@
+package display;
+
+import map.Map;
+
+import javax.swing.*;
+import java.awt.*;
+
+import static display.MapDisplay.imgHeight;
+import static display.MapDisplay.imgWidth;
+
+public class EditorDisplay extends JPanel {
+
+    private JPanel container = new JPanel();
+    public static float scale = 1;
+
+    //w sumie wywolujemy to tylko raz, troche dziwne ze nie jest to w konstruktorze display mapy
+    public EditorDisplay(Map map) {
```

Comment:
Musimy wywołać ten kod za każdym razem, kiedy zmienią się kafelki na planszy, np. jak użytkownik postawi nowe drzwi. To nie może być w konstruktorze, bo wtedy musimy tworzyć nowy obiekt za każdym razem, jak chcemy coś zmienić na planszy.